tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Bootstraps unit-tests.
+ *
+ * @package GlotPress
+ * @subpackage Tests
+ */
+
+if ( ! defined( 'GP_TESTS_DIR' ) ) {
+	define( 'GP_TESTS_DIR', __DIR__ );
+}
+
+if ( ! defined( 'GP_DIR_TESTDATA' ) ) {
+	define( 'GP_DIR_TESTDATA', GP_TESTS_DIR . '/data' );
+}
+
+if ( ! defined( 'GP_TESTS_PERMALINK_STRUCTURE' ) ) {
+	define( 'GP_TESTS_PERMALINK_STRUCTURE', '/%postname%' );
+}
+
+if ( ! defined( 'GP_TESTS_PERMALINK_STRUCTURE_WITH_TRAILING_SLASH' ) ) {
+	define( 'GP_TESTS_PERMALINK_STRUCTURE_WITH_TRAILING_SLASH', '/%postname%/' );
+}
+
+$_tests_dir = getenv( 'WP_TESTS_DIR' );
+
+if ( ! $_tests_dir ) {
+	$_tests_dir = rtrim( sys_get_temp_dir(), '/\\' );
+}
+$_tests_dir .= '/wordpress-tests-lib';
```

Comment:
I use the `WP_TESTS_DIR` and [it's supposed to include the `wordpress-tests-lib` path](https://github.com/GlotPress/gp-translation-helpers/blob/535de7f17c9c49a0bf3e62604dd5d4fdaa0e90da/tests/phpunit/bin/install-wp-tests.sh#L14), thus it should be only added if the `WP_TESTS_DIR` constant is not being used.

```suggestion
if ( ! $_tests_dir ) {
	$_tests_dir = rtrim( sys_get_temp_dir(), '/\\' ) . '/wordpress-tests-lib';
}
```